catalog/transformer/catalog-transformer-generic/src/main/java/ddf/catalog/transformer/generic/xml/SaxEventHandlerFactory.java

```diff
@@ -0,0 +1,9 @@
+package ddf.catalog.transformer.generic.xml;
+
+import ddf.catalog.util.Describable;
+
+public interface SaxEventHandlerFactory extends Describable {
```

Comment:
Is this api being exposed for implementation? If so, it should be in a separate api bundle and not in this implementation bundle. I'm asking because it's not clear if this interface is here for that purpose or not.